connector/cargo_template/Cargo.toml

```diff
@@ -7,4 +7,4 @@ edition = "2021"
 [dependencies]
 fluvio = { git = "https://github.com/infinyon/fluvio", rev = "{{fluvio-cargo-dependency-hash}}" }
 fluvio-connector-common = { git = "https://github.com/infinyon/fluvio", rev = "{{fluvio-cargo-dependency-hash}}", features = ["derive"]}
-serde = { version = "1.0", default-features = false, features = ["derive"]}
+serde = { default-features = false, features = ["derive"], workspace = true }
```

Comment:
templates can't have workspace dependency since they will be deployed outside fluvio repo

Reply:
Oopss... Fixed!